packages/ui/src/components/Panel.svelte

```diff
@@ -33,32 +34,34 @@
 </script>
 
 <div class="popupPanel" bind:clientWidth={panelWidth}>
-  <div class="popupPanel-title">
-    <Button
-      icon={IconClose}
-      kind={'transparent'}
-      size={'medium'}
-      on:click={() => {
-        dispatch('close')
-      }}
-    />
-    <div class="popupPanel-title__content"><slot name="title" /></div>
-    <div class="buttons-group xsmall-gap">
-      <slot name="utils" />
-      {#if asideFloat && $$slots.aside && isAside}
-        {#if $$slots.utils}<div class="buttons-divider" />{/if}
-        <Button
-          icon={IconDetails}
-          kind={'transparent'}
-          size={'medium'}
-          selected={asideShown}
-          on:click={() => {
-            asideShown = !asideShown
-          }}
-        />
-      {/if}
+  {#if isTitle}
```

Comment:
I personally think we should have same header for both modes. So edit mode for us should just switch between title/description editor and not touch right part with status components.